help/blocks/accordion/accordion.css

```diff
@@ -5,15 +5,7 @@
     background-color: var(--background-color);
   }
 
-  .accordion-wrapper:has(.content-center) {
```

Comment:
somehow these are not required now as per their website. I had added these CSS basis theie website when I implemented it. Weird but let's remove these now.